### ISSUE TITLE
WCAG 2.1 error: Variation tables on product pages are layout tables without role="presentation" attribute

### ISSUE THREAD

**Issue by melek** [woothemes/woocommerce#30803]
**Prerequisites (mark completed items with an [x]):**
- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**
According to WCAG 2.1, "Layout Tables" should have `role=presentation` in the table tag, but on variable product pages the Variations selectors are in tables without this tag. (If they were tabular data, they should use `th` tags)

According to [WAVE Webaim](https://wave.webaim.org/), if a table is being used for layout rather than tabular data:

>  If the layout table remains, verify that the reading and navigation order of table content (based on underlying source code order) is logical and give it role="presentation" to ensure it is not identified as a table to screen reader users.

**Expected behavior**
Variable product pages should not throw a WCAG warning/error due to a missing role attribute.

**Actual behavior**
A warning appears.

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. In Storefront (or any theme that doesn't override the Variations table), view any variable product page.  
Quick link: https://wave.webaim.org/report#/https://themes.woocommerce.com/storefront/product/checked-hoodie/
2. Inspect the Variations input table.
3. See error: `<table ... >` tag doesn't have a `role="presentation"` attribute.
4. Enter the product page URL into https://wave.webaim.org/ to confirm warning (or use quick link above)

**Screenshots**

![Screenshot](https://d.pr/i/5IHIMC+.png)  
*If you do not see the image, please [click here](https://d.pr/i/5IHIMC).*

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Additional Info**
I think changing the markup to use `div` for pure layout or `th` tags to convert to tabular data isn't quite right, and too disruptive to all the customizations that are likely out there, so I believe the `role` attribute is the best way to approach this since it only adds an attribute without any breaking markup changes.

* Related to #24185 and #28837
* 4321834-zen

**Comment by rodelgc** [comment]
Hi @melek ,

Thank you for taking the time to report this bug, we really appreciate your help. I can indeed see that `role=presentation` is not present in the `table` element for variation products. Tested in WC 5.6.0:

<img width="1452" alt="Screen Shot 2021-09-24 at 5 16 53 PM" src="https://user-images.githubusercontent.com/4509348/134650773-c66b92d3-6bc0-4072-a48e-65eb7303e2ec.png">


Priority of this issue will be assigned shortly after we'd go through a round of prioritization. Please note it may take a couple of days.
